code/style.css

```diff
@@ -1,3 +1,98 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@300&family=Roboto:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
```

Comment:
A, you put fonts in the css instead of the html! Is that better in some way? Curious!